modules/backend/layouts/auth.htm

```diff
@@ -10,6 +10,29 @@
         <link rel="icon" type="image/png" href="<?= e(Backend\Models\BrandSetting::getFavicon()) ?>">
         <title><?= e(trans('backend::lang.auth.title')) ?></title>
 
+        <!-- Inline Javascript to reduce lookups -->
+        <script>
+        // Load the Preload Assets	
+        $(document).ready(function(){
```

Comment:
@ayumihamsaki you could either specify filed as `autofocus` (https://www.w3schools.com/tags/att_input_autofocus.asp) or with tabindex='0' (old school style https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex)

Reply:
@w20k I'm confused by your comment, you are talking about form fields ?

Reply:
@ayumihamsaki I'm talking about this jQuery block, which ain't needed from my point of view. Or I'm not seeing the whole picture here.

Reply:
Will add it to my list of ToDo, could be done differently :)

Reply:
Let me know, this pr is missing the other extra code I did in this pr: https://github.com/octobercms/october/pull/4285

It would be nice to combine both PR's and have a finished merged code.

Reply:
@ayumihamsaki sorry about that! Will merge it tomorrow everything is good there need to run a smoketest. That's all! Thanks for your patience ;)

Reply:
@w20k Take your time, no need to rush. If you want to do it next week, no problem! I am always around ... p.s. Thanks for your help always!